components/status-indicator/README.md

```diff
@@ -65,6 +65,139 @@ Status Indicators are used to communicate the status of an item. They are non-in
+    <d2l-status-indicator state="alert" text="overdue"></d2l-status-indicator>
+    <d2l-status-indicator state="none" text="closed"></d2l-status-indicator>
+  </div>
+
+  <div class="status-format">
+    <d2l-status-indicator state="default" text="due today" bold></d2l-status-indicator>
+    <d2l-status-indicator state="success" text="complete" bold></d2l-status-indicator>
+    <d2l-status-indicator state="alert" text="overdue" bold></d2l-status-indicator>
+    <d2l-status-indicator state="none" text="closed" bold></d2l-status-indicator>
+  </div>
+</div>
+```
+The state is used to apply a meaningful colour to the status indicator to assist with scannability. The visible label should still have a clear meaning so that users who can't see the colour can still understand the state.
+
+* Default - the state is important but not urgent 
+* Success - the state is considered positive or complete.
+* Alert - the state requires urgent attention
+* None - the state is not important
+
+### Content
+
+<!-- docs: demo -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/status-indicator/status-indicator.js';
+  import '@brightspace-ui/core/components/icons/icon.js';
+</script>
+<style>
+	.align-item {
+		color: var(--d2l-color-tungsten);
+		font-size: 0.7rem;
+	}
+	.align-item.bullet::before {
+		color: var(--d2l-color-tungsten);
+		content: '\002022';
+		margin-left: 5px;
+		margin-right: 5px;
+	}
+	.card {
+		min-width: 300px
+	}
+	.icon {
+		width: 60px;
+	}
+	.row {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: nowrap;
+		width: 100%;
+	}
+	.column {
+		display: flex;
+		flex-direction: column;
+		flex-basis: 100%;
+		flex: 1;
+	}
+	.indicator {
+		margin-bottom: 12px;
+		min-width: 350px;
+	}
+	d2l-icon {
+		margin-right: 15px;
+		height: 45px;
+		min-width: 45px;
+	}
+	.text {
+		font-size: 16px;
+		line-height: 18px;
+	}
+</style>
+<div class="row">
+
+	<d2l-icon icon="tier3:assignments">
+	</d2l-icon>
+
+	<div class="column">
+		<h4 style="margin: 0;">
+			Hurricanes - what, where, why
+		</h4>
+		<div class="indicator">
+			<d2l-status-indicator state="alert" text="overdue" bold></d2l-status-indicator>
+			<span class="align-item" style="margin-left: 10px;">Grade 6 Science</span>
+			<span class="align-item bullet">Assignment</span>
+		</div>
+		<div class="d2l-typography text">
+			Provide a detailed description of the information that you received in our lesson on how hurricanes are formed and where they are most likely to occur.
+		</div>
+	</div>
+
+</div>
+```
+
+The text label should be kept short; one or two words at most. If more information is needed to explain the state or give extra context, the information should be included after the label and should use the same color to associate the information with the state. When placing additional text on the same line as the status indicator, ensure that the bottom of the text is even across all elements.
+
 <!-- docs: start hidden content -->
 ### Variants
```

Comment:
Should we get rid of this section given that the one above explains it so it just adds duplication?